doc/source/ray-overview/installation.rst

```diff
@@ -140,8 +140,6 @@ You can install the nightly Ray wheels via the following links. These daily rele
 
   :ref:`Usage stats <ref-usage-stats>` collection is enabled by default (can be :ref:`disabled <usage-disable>`) for nightly wheels including both local clusters started via ``ray.init()`` and remote clusters via cli.
 
-.. note::
-
   .. If you change the list of wheel links below, remember to update `get_wheel_filename()` in  `https://github.com/ray-project/ray/blob/master/python/ray/_private/utils.py`.
```

Comment:
perhaps remove this line too, it doesn't show up (commented out?)

Reply:
Yeah, this looks like a comment for people editing `installation.rst` and still seems relevant (in fact it seems this needs to be updated: https://github.com/ray-project/ray/blob/681dd0e9cc98c4ccd5455e23f8492de034507fb2/python/ray/_private/utils.py#L1225), so it is probably better to leave it :)